### ISSUE TITLE
More precise temperature

### ISSUE THREAD

**Issue by RuslanasMobOn** [davemckelvie/attiny-bus-buddy#2]
Hi, is there a possibility for quick fix to read more precise temperature from DS18B20? For now i'm getting just integer. It would be great to get double value.

**Comment by davemckelvie** [comment]
Yep, can be done. Probably the easiest way is to return the raw value from the DS18B20 as two bytes then do the temperature conversion in the I2C master.

**Comment by RuslanasMobOn** [comment]
I'm not that much familiar with c/c++ and converting raw data to data types (even more i don't know how to debug ATTiny with Platformio) . It would be great if you give me more clearer hints

**Comment by davemckelvie** [comment]
what are you trying to achieve? what can you tell me about your project?

**Comment by RuslanasMobOn** [comment]
I have raspberry pi with android things, i need to get temperature from my waterproof ds18b20, so i read your article, bought ATTiny85 and now i’m getting temperature via ATTiny as slave. But temperature is two digit integer, but i want to get float with more precise value. I’m not familiar with c/c++ , i just use your code and diagrams. So i will be greatful if you help me to get that float value from ds18b20
